README.md

```diff
@@ -1,3 +1,18 @@
+---
+
+This library, ADAL for JavaScript, will no longer receive new feature improvements. Instead, use the new library
+[MSAL.js](https://github.com/AzureAD/microsoft-authentication-library-for-js).
+
+* If you are starting a new project, you can get started with the
+  [MSAL.js docs](https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki)
+  for details about the scenarios, usage, and relevant concepts.
+* If your application is using the previous ADAL JavaScript library, you can follow this
+  [migration guide](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-compare-msal-js-and-adal-js)
```

Comment:
nit: need to delete "en-us" in this link so it will localize

Reply:
Updated, although I'll need to follow up with some other folks. I heard that the loc issue would be fixed soon, so maybe we'll be saved automagically!